vwifi.c

```diff
@@ -37,6 +37,7 @@ struct owl_context {
     struct mutex lock;
     struct work_struct ws_connect, ws_disconnect;
     char connecting_ssid[SSID_MAX_LENGTH + 1]; /* plus one for '\0' */
+    char ssid[SSID_MAX_LENGTH + 1];            /* SSID for AP mode */
```

Comment:
The array size should be `SSID_MAX_LENGTH`, since IEEE 802.11 SSID don't need a NULL terminator. Every real driver I have read have the capacity of 32 bytes for storing SSID rather than 33 bytes.